src/modules/createTasks.js

```diff
@@ -0,0 +1,107 @@
+const toDoList = document.querySelector('#todo-lists');
+const clearBtn = document.querySelector('button');
+
+export default class CreateTasks {
+    static createTask = () => {
+      const task = document.createElement('div');
+      task.className = 'todo-list-container';
+      toDoList.appendChild(task);
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      const content = document.createElement('p');
+      content.className = 'content';
+      const dots = document.createElement('i');
+      dots.className = 'dots fa-solid fa-ellipsis-vertical';
+      const trash = document.createElement('i');
+      trash.className = 'alt-fa fa-solid fa-trash';
+
+      task.append(checkbox, content, dots, trash);
+
+      checkbox.addEventListener('click', () => {
+        task.classList.toggle('checked-holder');
```

Comment:
[OPTIONAL]
- [ ] It is required to listen for a `change` rather than a `click` event on the checkbox.